guides/predictive-analytics-with-deepnote.md

```diff
@@ -0,0 +1,124 @@
+
+  * 'Years of experience'
+  * 'Salary'
+
+Your datasets will likely differ in scope and complexity, but this will hopefully serve as a general tutorial on how to leverage your data with Hydra & Deepnote for predictive analytics.
+
+
+## Setup
+
+* Ensure you have your Hydra login details handy or follow the [Setup guide](https://docs.hydra.so/getting-started/setup-guide) to create your initial Hydra data warehouse.
+
+* Sign up for your free instance at [Deepnote](https://deepnote.com/sign-up)
+
+### Building the table with Hydra
+
+1. Sample data from [Kaggle](https://www.kaggle.com/datasets/rsadiq/salary) would be used for this guide. Download the CSV file.
+2. Connect to your Hydra database and import the CSV file to the database using the steps for importing a CSV file outlined in the [Documentation](https://docs.hydra.so/centralize-data/load/from-local-csv-file)
+
+### Creating the Connection
+
+Now we can connect Deepnote to our sample data in Hydra.
+
+1. Start by [signing up](https://deepnote.com/sign-up) for Deepnote if you haven't done so already.
+2. Within your Deepnote account, create a workspace name. In the case of this tutorial, the workspace name will be "Hydra-Team".
+3. Next, choose 'PostgreSQL' as the data source as shown in the image below.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/datasource.png" alt=""><figcaption><p>Choosing Datasource</p></figcaption></figure>
+
+4. Create a new project by clicking on '+' under 'Projects' within the left panel of the Deepnote console.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/create project.png" alt=""><figcaption><p>Creating project</p></figcaption></figure>
+
+5. Within your new project, navigate to 'Integrations' on the right panel and choose 'PostgreSQL'.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/Integrating postgres.png" alt=""><figcaption></figcaption></figure>
+
+6. At the 'Connect to PostgresSQL' popup, add your Hydra database credentials including Hostname, User, Password, and Database.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/connecting postgres.png" alt=""><figcaption><p>Connecting PostgreSQL</p></figcaption></figure>
+
+7. After establishing the connection, you can view the imported salary table within Deepnote.
+
+Using the SELECT statement, you are able to run SQL queries on the salary table which enables you to retrieve and analyze specific data from the table.
+
+```sql
+SELECT * FROM public.salary LIMIT 100
+```
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/deepnote dashboard.png" alt=""><figcaption><p>Deepnote dashboard showing the salary table</p></figcaption></figure>
+
+8. To access the table using Python scripts because predictions can not be done using SQL, an engine has to be created with the data warehouse credentials:
```

Comment:
```suggestion
8. To access the table using Python scripts because predictions can not be done using SQL, an engine has to be created with the data warehouse credentials. Replace the credential, table name, and schema as appropriate in the example script below:
```